src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -68,14 +72,29 @@ public void Loaded(ViewLoadedParams viewLoadedParams)
             this.documentationBrowserMenuItem.Unchecked += MenuItemUnCheckedHandler;
             this.viewLoadedParamsReference.AddMenuItem(MenuBarType.View, this.documentationBrowserMenuItem);
 
-
             DynamoView.CloseExtension += OnCloseExtension;
 
             // subscribe to the documentation open request event from Dynamo
             this.viewLoadedParamsReference.RequestOpenDocumentationLink += HandleRequestOpenDocumentationLink;
 
             // subscribe to property changes of DynamoViewModel so we can show/hide the browser on StartPage display
             (viewLoadedParams.DynamoWindow.DataContext as DynamoViewModel).PropertyChanged += HandleStartPageVisibilityChange;
+
+            // subscribe to package loaded so we can add the package documentation 
+            // to the Package documentation manager when a package is loaded
+            pmExtension.PackageLoader.PackgeLoaded += OnPackgeLoaded;
```

Comment:
it's possible pmExtension is null.

Reply:
Seems we also forgot to unsubscribe this during disposing